core/domain/android_validation_constants.py

```diff
@@ -23,14 +23,14 @@
 # changed, just this array has to be modified to include the new
 # interactions or remove existing ones.
 VALID_INTERACTION_IDS = [
-    'Continue', 'EndExploration', 'NumericInput', 'TextInput',
-    'FractionInput', 'NumberWithUnits', 'MultipleChoiceInput',
-    'ItemSelectionInput']
+    'Continue', 'DragAndDropSortInput', 'EndExploration', 'FractionInput',
```

Comment:
This isn't true right now. The Android app does not yet support drag-and-drop support (we're currently in the process of adding support for it).

Reply:
Ah I see. The reason I'm enabling this now is to unblock publishing of stories that contain lessons with the drag-and-drop interaction.

@seanlip do you think we can wait until android adds support for this?

Reply:
@BenHenning Can we still add this? I need it to publish the lesson in the first place.

Reply:
Okay, conditionally allowing this change since the Android app won't yet be majorly affected by it. Thanks for the additional context @seanlip!